code/game/gamemodes/blob/blobs/captured_nuke.dm

```diff
@@ -0,0 +1,28 @@
+/obj/structure/blob/captured_nuke //alternative to blob just straight up destroying nukes
+	name = "blob captured nuke"
+	icon = 'icons/mob/blob.dmi'
+	icon_state = "blob"
```

Comment:
is this necessary?

Reply:
Yep. Since `/obj/structure/blob` never defines `icon_state`